### PR TITLE
Adding subject line to the submit your pitch button on homepage

### LINKED ISSUE #1975: Change Submit your pitch button on homepage
### Overview
We need to have the submit your pitch button convey a subject line, so that when people click on it and send an email we can tie to the action of having clicked the button.

### Action Items
- [x] Add subject line to button action

Subject line should be:
Submitting my pitch to Hack for LA

### Resources/Instructions
Example: 
```
<a href="mailto:test@example.com?subject=Testing out mailto!">First Example</a>
```

### PR DESCRIPTION
**Pull request by SAUMILDHANKAR**
Fixes #1975 

### What changes did you make and why did you make them ?

  - Added the subject line to the submit your pitch button.
  - This was required as part of the issue.
  

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>No change in visuals before changes are applied</summary>

</details>

<details>
<summary>No change in visuals after changes are applied</summary>
  

</details>

### CODE REVIEW

**Review by Zhu-Joseph — DISMISSED** [review]
Clears issue

**Review by macho-catt — CHANGES_REQUESTED** [review]
Hi @SAUMILDHANKAR. Good job on properly checking the action items on the issue and detailing the changes you made and adding screenshots to the PR.

However, it looks like the 'ask' from the issue was different. In your PR, you changed the text of the button, but the ask was to change the subject line instead.

The example code shows how to do a subject line:

```
<a href="mailto:test@example.com?subject=Testing out mailto!">First Example</a>
```

So for this example, you don't change the title, which is "First Example." That should remain. What you need to change is "Testing out mailto," which is the subject line. So for your code, it should be something along the lines of adding a subject line in the href parameter of the a tag.

Let me know if you have any questions.

File: _includes/current-projects.html [hunk 1]
```diff
@@ -5,7 +5,7 @@ <h2 class="project-header">Current Projects</h2>
     <p class="project-pitch">
       Have an idea?
       <a href="mailto:team@hackforla.org" class="btn btn-primary btn-md-narrow"
-        >Submit your pitch</a
+        >Submitting my pitch to Hack for LA</a
```
Comment (macho-catt) [inline _includes/current-projects.html]: Revert this since the title of the button should stay the same

File: _includes/current-projects.html [hunk 2]
```diff
@@ -5,7 +5,7 @@ <h2 class="project-header">Current Projects</h2>
     <p class="project-pitch">
       Have an idea?
       <a href="mailto:team@hackforla.org" class="btn btn-primary btn-md-narrow"
```
Comment (macho-catt) [inline _includes/current-projects.html]: Subject line would go here. For example:
```
<a href="mailto:test@example.com?subject=Testing out mailto!">First Example</a>
```
Where the subject line is "Testing out mailto!"

Comment (SAUMILDHANKAR) [reply _includes/current-projects.html]: @macho-catt Thanks for your review and suggestions. I will make the changes and let you know if I have any more questions. Thanks.

**Comment by SAUMILDHANKAR** [comment]
@macho-catt I have made the changes as suggested in my latest commit. Please let me know if more changes are required before merging. Thank you.

**Review by macho-catt — APPROVED** [review]
Looks good to me. New subject line appears in my environment. Good job!

**Comment by SAUMILDHANKAR** [comment]
Thank you.

**Pull request merged**